internal/workflow/preprocessing.go

```diff
@@ -69,6 +70,34 @@ func (w *PreprocessingWorkflow) Execute(
 		&activities.ValidateStructureParams{SIP: identifySIP.SIP},
 	).Get(ctx, &validateStructure)
 
+	// Add PREMIS objects.
+	var addPREMISObjects activities.AddPREMISObjectsResult
+	e = temporalsdk_workflow.ExecuteActivity(
+		withLocalActOpts(ctx),
+		activities.AddPREMISObjectsName,
+		&activities.AddPREMISObjectsParams{
+			Path:        localPath,
+			ContentPath: identifySIP.SIP.ContentPath},
```

Comment:
Bracket. I'll stop flagging them, I hope they get fixed by linting, otherwise you get the point ;)